src/main/java/io/github/thebusybiscuit/slimefun4/core/services/holograms/Hologram.java

```diff
@@ -0,0 +1,136 @@
+package io.github.thebusybiscuit.slimefun4.core.services.holograms;
+
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import org.bukkit.Bukkit;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Entity;
+
+import com.google.common.base.Objects;
```

Comment:
The one in Java does the same, this has been deprecated since J7

Reply:
whoops, wrong import